app/views/admin/suppliers/index.html.haml

```diff
@@ -3,6 +3,18 @@
     %h1.govuk-heading-xl
       Suppliers
 
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    %form{method: 'get'}
+      %h2.govuk-heading-s
+        Search
+      .ccs-search-form-group
+        %label.govuk-label.govuk-visually-hidden{for: 'search'} Search
+        %input#search.govuk-input{name: 'search', type: 'text', value: params[:search]}
```

Comment:
What do you think about testing the behaviour that the search text is persistent after performing the search?

Reply:
I think it is a good idea, I'll give it a try.